webapp/src/main/webapp/resources/output.js

```diff
@@ -12,18 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-function unknownColumnWarning() {
-  // TODO: change from hardcoded to parameters
-  const params = {
-    numNotices: 1,
-    notices: [
-      {
-        filename: "hello.txt",
-        fieldName: "column name",
-        index: 5
-      }
-    ]
+function callCorrespondingFunction(noticesJSON) {
+  const noticeContainer = JSON.parse(noticesJSON);
+  for (var i = 0; i < noticeContainer.notices.length; i++) {
+    const notice = noticeContainer.notices[i];
+    // Notice has not been implemented, output raw json
+    if (!runFunctionName(notice.code, notice)) {
+      const unknownNotice = document.createElement('p');
+      unknownNotice.innerHTML = 'NOTICE CONTAINER NOT IMPLEMENTED - RAW JSON DATA: <br><br>' + JSON.stringify(notice);
+      document.getElementById("unimplementedNotices").appendChild(unknownNotice);
+    } else {
+      console.log("Notice has been handled!");
```

Comment:
Can we avoid writing to console during non error execution?

Reply:
Yep, will remove this.

Reply:
Done!